flang/test/Lower/intrinsic-procedures.f90

```diff
@@ -323,6 +340,60 @@ subroutine iand_test(a, b)
   ! CHECK: %{{[0-9]+}} = and %{{[0-9]+}}, %{{[0-9]+}} : i{{(8|16|32|64|128)}}
 end subroutine iand_test
 
+! IBCLR
+! CHECK-LABEL: ibclr_test
+function ibclr_test(i, j)
+  ! CHECK-DAG: %[[result:.*]] = fir.alloca i32 {bindc_name = "ibclr_test"
+  ! CHECK-DAG: %[[i:.*]] = fir.load %arg0 : !fir.ref<i32>
```

Comment:
We're safe for now assuming MLIR will pretty-print the argument values as `%arg0`, `%arg1`, ... but that convention tends towards "happy accident" as MLIR could change it at any time.